src/hooks/useMutation.ts

```diff
@@ -20,6 +20,7 @@ export type UseMutationResponse<T, V> = [
 export const useMutation = <T = any, V = object>(
   query: DocumentNode | string
 ): UseMutationResponse<T, V> => {
+  const devtoolsContext = useRef({ source: getHookParent() });
```

Comment:
Can we maybe make this a hook that just returns a ref; or just returns the value itself if that's possible and stable?